src/core/src/op/reduce_prod.cpp

```diff
@@ -99,9 +99,7 @@ bool ReduceProd::evaluate_upper(ov::TensorVector& output_values) const {
     // In case dimensions has a zero dimension - it should return 0 in any case
     if (tensor_has_max_value(get_input_tensor(0).get_upper_value()) &&
         !tensor_has_zero_value(get_input_tensor(0).get_upper_value())) {
-        OPENVINO_SUPPRESS_DEPRECATED_START
-        auto max_constant = ngraph::get_constant_max_of_type(get_output_element_type(0));
-        OPENVINO_SUPPRESS_DEPRECATED_END
+        auto max_constant = ov::util::get_constant_max_of_type(get_output_element_type(0));
```

Comment:
Create tensor instead constant and copy data between tensors.

Reply:
Done